### PR TITLE
Cleanup and macros

### PR DESCRIPTION
**Pull request by Soldia1138**
**When merged this pull request will:**
- Make Jonpas happy
- Solve open Points in #206

### CODE REVIEW

File: addons/sys_prc117f/CfgAcreRadios.hpp [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
         sinadRating = -118;
         sensitivityMin = -118;
         sensitivityMax = -50;
         isPackRadio = 1;
         isDeployable = 0;
 
-        connectors[] = {
-                            {"Antenna", ACRE_CONNECTOR_TNC},
-                            {"Audio/Data", ACRE_CONNECTOR_U_283}
+        connectors[] = { {"Antenna", ACRE_CONNECTOR_TNC}, {"Audio/Data", ACRE_CONNECTOR_U_283}
```
Comment (jonpas) [inline addons/sys_prc117f/CfgAcreRadios.hpp]: Split into separate lines, like that:
```
connectors[] = {
    {"Antenna", ACRE_CONNECTOR_TNC},
    {"Audio/Data", ACRE_CONNECTOR_U_283}
};
```

File: addons/sys_prc117f/CfgAcreRadios.hpp [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
                         };
-        defaultComponents[] = {
-                                {0, "ACRE_120CM_VHF_TNC"},
-                                {1, "ACRE_14IN_UHF_TNC"}
+        defaultComponents[] = { {0, "ACRE_120CM_VHF_TNC"}, {1, "ACRE_14IN_UHF_TNC"}
```
Comment (jonpas) [inline addons/sys_prc117f/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_prc148/CfgAcreRadios.hpp [hunk 3]
```diff
@@ -19,13 +18,10 @@ class CfgAcreComponents
          Array of arrays, each item being {connectorLabel, connectorType}
         */
 
-        connectors[] = {
-                            {"Antenna", ACRE_CONNECTOR_TNC},
-                            {"Audio/Data", ACRE_CONNECTOR_U_283}
+        connectors[] = { {"Antenna", ACRE_CONNECTOR_TNC}, {"Audio/Data", ACRE_CONNECTOR_U_283}
```
Comment (jonpas) [inline addons/sys_prc148/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_prc148/CfgAcreRadios.hpp [hunk 4]
```diff
@@ -19,13 +18,10 @@ class CfgAcreComponents
...
                         };
 
-        defaultComponents[] = {
-                {0, "ACRE_120CM_VHF_TNC"}
+        defaultComponents[] = { {0, "ACRE_120CM_VHF_TNC"}
```
Comment (jonpas) [inline addons/sys_prc148/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_prc152/CfgAcreRadios.hpp [hunk 5]
```diff
@@ -1,26 +1,22 @@
...
         isDeployable = 0;
 
-        connectors[] = {
-                            {"Antenna", ACRE_CONNECTOR_TNC},
-                            {"Audio/Data", ACRE_CONNECTOR_U_283}
+        connectors[] = { {"Antenna", ACRE_CONNECTOR_TNC}, {"Audio/Data", ACRE_CONNECTOR_U_283}
                         };
-        defaultComponents[] = {
-                                {0, "ACRE_120CM_VHF_TNC"}
+        defaultComponents[] = { {0, "ACRE_120CM_VHF_TNC"}
```
Comment (jonpas) [inline addons/sys_prc152/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_prc152/CfgAcreRadios.hpp [hunk 6]
```diff
@@ -1,26 +1,22 @@
...
         sinadRating = -118;
         sensitivityMin = -118;
         sensitivityMax = -50;
         isPackRadio = 0;
         isDeployable = 0;
 
-        connectors[] = {
-                            {"Antenna", ACRE_CONNECTOR_TNC},
-                            {"Audio/Data", ACRE_CONNECTOR_U_283}
+        connectors[] = { {"Antenna", ACRE_CONNECTOR_TNC}, {"Audio/Data", ACRE_CONNECTOR_U_283}
```
Comment (jonpas) [inline addons/sys_prc152/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_prc343/CfgAcreRadios.hpp [hunk 7]
```diff
@@ -1,26 +1,22 @@
...
         sinadRating = -100;
         sensitivityMin = -110;
         sensitivityMax = -50;
         isPackRadio = 0;
         isDeployable = 0;
 
-        connectors[] = {
-                            {"Antenna", ACRE_CONNECTOR_TNC},
-                            {"Audio/Data", ACRE_CONNECTOR_U_283}
+        connectors[] = { {"Antenna", ACRE_CONNECTOR_TNC}, {"Audio/Data", ACRE_CONNECTOR_U_283}
```
Comment (jonpas) [inline addons/sys_prc343/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_prc343/CfgAcreRadios.hpp [hunk 8]
```diff
@@ -1,26 +1,22 @@
...
                         };
-        defaultComponents[] = {
-                                {0, "ACRE_2HALFINCH_UHF_TNC"}
+        defaultComponents[] = { {0, "ACRE_2HALFINCH_UHF_TNC"}
```
Comment (jonpas) [inline addons/sys_prc343/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_prc77/CfgAcreRadios.hpp [hunk 9]
```diff
@@ -1,26 +1,22 @@
...
         sinadRating = -118;
         sensitivityMin = -118;
         sensitivityMax = -50;
         isPackRadio = 1;
         isDeployable = 0;
 
-        connectors[] = {
-                            {"Antenna", ACRE_CONNECTOR_TNC},
-                            {"Audio/Data", ACRE_CONNECTOR_U_283}
+        connectors[] = { {"Antenna", ACRE_CONNECTOR_TNC}, {"Audio/Data", ACRE_CONNECTOR_U_283}
```
Comment (jonpas) [inline addons/sys_prc77/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_prc77/CfgAcreRadios.hpp [hunk 10]
```diff
@@ -1,26 +1,22 @@
...
                         };
-        defaultComponents[] = {
-                                { 0,"ACRE_120CM_VHF_TNC" }
+        defaultComponents[] = { { 0,"ACRE_120CM_VHF_TNC" }
```
Comment (jonpas) [inline addons/sys_prc77/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_sem52sl/CfgAcreRadios.hpp [hunk 11]
```diff
@@ -1,26 +1,22 @@
...
         sinadRating = -117;
         sensitivityMin = -117;
         sensitivityMax = -50;
         isPackRadio = 0;
         isDeployable = 0;
 
-        connectors[] = {
-                            {"Antenna", ACRE_CONNECTOR_BNC},
-                            {"Audio/Data", ACRE_CONNECTOR_U_283}
+        connectors[] = { {"Antenna", ACRE_CONNECTOR_BNC}, {"Audio/Data", ACRE_CONNECTOR_U_283}
```
Comment (jonpas) [inline addons/sys_sem52sl/CfgAcreRadios.hpp]: Same as above.

File: addons/sys_sem52sl/CfgAcreRadios.hpp [hunk 12]
```diff
@@ -1,26 +1,22 @@
...
                         };
-        defaultComponents[] = {
-                                {0, "ACRE_120CM_VHF_BNC"} /*TODO Not use this */
+        defaultComponents[] = { {0, "ACRE_120CM_VHF_BNC"} /*TODO Not use this */
```
Comment (jonpas) [inline addons/sys_sem52sl/CfgAcreRadios.hpp]: Same as above.

**Review by jonpas — CHANGES_REQUESTED** [review]
Why are the TRACEs getting removed?

**Comment by Soldia1138** [comment]
The Traces were only used during development of the operating system on 152 and 117. They are pretty much useless for debugging (and were commented out for exactly that reason)

**Review by jonpas — APPROVED** [review]

**Comment by jonpas** [comment]
I see! 👍

**Pull request merged**